open-metadata-implementation/user-interfaces/view-services/type-explorer-view/type-explorer-view-spring/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>type-explorer-view</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.3-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>type-explorer-view-spring</artifactId>
+    <dependencies>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>type-explorer-view-server</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>type-explorer-view-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.tomcat.embed</groupId>
+            <artifactId>tomcat-embed-core</artifactId>
```

Comment:
as above - though I'm not familar enough with spring. Will experiment when the code is buildable pr

Reply:
shouldn't depend on tomcat, as suppose to run in different servlet container also

Reply:
Any recommendations on how to resolve @bogdan-sava ? 
I'm not familar enough with this area
How about we open up a new issue to track so that we can get the big merge in? Does that work?

Reply:
reopening ^^^

Reply:
Ok I've opened an issue for us to discuss/track. Allowing us to do the merge. See https://github.com/odpi/egeria/issues/2267